src/formio/components/AddressNL.scss

```
@@ -0,0 +1,19 @@
+@import '~microscope-sass/lib/responsive';
+
+.openforms-addressnl {
+  display: grid;
+  grid-column-gap: var(--of-addressnl-column-gap, 8px);
+  grid-row-gap: var(--of-addressnl-row-gap, var(--of-form-field-container-gap, 24px));
+}
+
+.openforms-addressnl.openforms-addressnl--single-column {
+  grid-template-columns: 1fr;
+}
+
+.openforms-addressnl.openforms-addressnl--double-column {
+  grid-template-columns: 1fr 1fr;
+
+  @include mobile-only {
+    grid-template-columns: 1fr;
+  }
+}
```

```suggestion
.openforms-addressnl {
  display: grid;
  grid-template-columns: var(--_of-addressnl-grid-template-columns, 1fr);
  grid-column-gap: var(--of-addressnl-column-gap, 8px);
  grid-row-gap: var(--of-addressnl-row-gap, var(--of-form-field-container-gap, 24px));
   
  @include bem.modifier('double-column') {
    --_of-addressnl-grid-template-columns: 1fr 1fr;
    
      @include mobile-only {
        grid-template-columns: 1fr;
      }
  }
}
```

* the `bem.modifier` makes sure to output the right class names
* the `grid-template-columns` variable approach is untested but I think it will work - the idea is that just override the variable for the more specific selector (and the default is single-column).